language/move-vm/runtime/src/loader.rs

```diff
@@ -1344,7 +1345,7 @@ impl<'a> Resolver<'a> {
 pub(crate) struct Module {
     id: ModuleId,
     // primitive pools
-    module: CompiledModule,
+    module: Arc<CompiledModule>,
```

Comment:
My only questions is whether this is likely to have unpleasant perf implications.

Reply:
I don't think so. `Module` is already wrapped in an `Arc`, if anything having this boxed might improve things... as the `Module` size is significantly smaller